cosmwasm/ibc-union/core/light-client-interface/src/lib.rs

```diff
@@ -170,6 +184,50 @@ fn client_impl<T: IbcClient>(
     Ok(addr)
 }
 
+pub struct StateUpdate<T: IbcClient> {
+    pub height: u64,
+    pub client_state: Option<T::ClientState>,
+    pub consensus_state: T::ConsensusState,
+    pub storage_writes: StorageWrites,
+}
+
+/// Client creation output type
+pub struct ClientCreation<T: IbcClient> {
+    /// The client state that is going to be stored by IBC. If set to `None`, IBC will store the
+    /// client state given by the creator as is
+    pub client_state: Option<T::ClientState>,
```

Comment:
what is this used for?

Reply:
For example in the case of Ethereum light client, `initial_sync_committee` exists under `ClientState`. But this is only to provide the sync committee at the initial state. Ethereum light client then sets this to `None` and returns the modified version of the client state so that IBC can commit that instead of committed the one that it gets from the relayer.

Reply:
why not just store the original state? is the `initial_sync_committee` large?

Reply:
It's total of 1024 bls pubkeys so 1024 * 384 bytes

Reply:
oof yeah lets zero that out indeed lol

Reply:
maybe leave a comment as to why we remove it